generators/app/templates/src/app/core/shell/_shell.component.scss

```diff
@@ -1,4 +1,38 @@
 <% if (props.ui === 'ionic') { -%>
+<%   if (props.layout === 'tabs') { -%>
+$tabs-height: 8.2rem;
```

Comment:
isn't there a ionic var for this? I think it's not the same depending of ios/android/wp

Reply:
there are the ionic vars: 

`$tabs-ios-tab-min-height` | 49px |  
`$tabs-md-tab-min-height` | 5.6rem |  
`$tabs-wp-tab-min-height` | 4.8rem |

Reply:
and the height is only for the vertical tabs, which must be greater than the `min-height` value